lib/Connection.php

```diff
@@ -30,6 +30,8 @@ class Connection {
     private $uri;
     /** @var int */
     private $timeout = 1000;
+    /** @var int */
+    private $persistent = 0;
```

Comment:
At the begining, I wish to set the flags. You give me a good suggest, it should only check the persistent.